Intro Skip/Displays/AnimatedSkipDisplayService.cs

```diff
@@ -43,6 +43,8 @@ public void Show()
                 textTransform.sizeDelta = new Vector2(100, 20);
                 _skipPromptText.fontSize = 15f;
                 canvas.enabled = true;
+
+                CameraUtils.Core.VisibilityUtils.SetLayer(_skipPromptObject, CameraUtils.Core.VisibilityLayer.UI);
```

Comment:
I'd rather not have an additional dependency for a one liner which itself only [invokes a one liner](https://github.com/Reezonate/CameraUtils/blob/bc91164a739e8c0bcd99a3d9fd259d03c605f3ca/Source/Core/VisibilityUtils.cs#L10), so I can't accept this as is.